static/app/views/organizationStats/usageStatsOrg.tsx

```diff
@@ -608,3 +610,35 @@ const FooterDate = styled('div')`
     font-size: ${p => p.theme.fontSizeMedium};
   }
 `;
+
+type SpansStoredProps = {
+  acceptedStored: string;
+  organization: Organization;
+};
+
+const StyledSettingsButton = styled(LinkButton)`
+  margin-top: -${space(0.5)};
+  padding: 3px;
+  border-radius: 50%;
+  width: 24px;
+  height: 24px;
```

Comment:
Let's not override the style of our default buttons if possible.

Reply:
Done with [00bd043](https://github.com/getsentry/sentry/pull/81311/commits/00bd043c23d278f8376da3d5c7fa11003ebb2ae7)